src/components/search/GlobalSearchField.js

```diff
@@ -210,9 +275,8 @@ function GlobalSearchField(props) {
     const router = useRouter();
     const { search, showErrorAnnouncer } = props;
 
-    const { t } = useTranslation(["common", "analyses"]);
-
-    const appRecordFields = appFields();
+    const { t } = useTranslation(["common", "analyses", "apps"]);
+    const appRecordFields = appFields(t);
```

Comment:
I thought this would only work if the keys in `appFields` used the namespace prefix, like `appsI18n("apps:name")`, `appsI18n("apps:rating")`, etc.

Reply:
✅